lib/biran/erb_config.rb

```diff
@@ -27,7 +27,7 @@ def process_erb
 
     def build_erb_env
       proc do
-        @environment = Configurinator.env
+        @environment = config[:env]
```

Comment:
isn't config a `Configurinator` instance? should it be `config.env`? or does this comes from the yaml parsed config?

Reply:
If I'm keeping everything straight, config here is the instance variable in the ERBConfig instance which is just a hash passed in and not an instance object. The configurinator instance builds the Configurinator.config hash and passes it to an instance of this class to process the templates. There is also a class method in configurinator that loads an object built from all the defaults and biran configuration (! app_config.yml, but initializers and env values). So as far as I know this should be fine and I don't think `config.env` even exists here because the instance var config is a hash versus an instance object of configurinator or configure.

Reply:
OK. Right, this is done through binding. I get it now.